### PR TITLE
[NOD-747] Change FinalityInterval to be 24 hours, isCurrent to be true if the DAG's time is less than 12 hours than the present, and change MaxInvPerMsg to be 1 << 17

### PR DESCRIPTION
**Pull request by someone235**

### CODE REVIEW

File: blockdag/dag.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ const (
 	// maxOrphanBlocks is the maximum number of orphan blocks that can be
 	// queued.
 	maxOrphanBlocks = 100
+
+	isDAGCurrentMaxDiff = 12 * time.Hour
```
Comment (stasatdaglabs) [inline blockdag/dag.go]: TargetTimePerBlock and FinalityInterval are in dagparams. I argue that this should be in dagparams too.

**Comment by someone235** [comment]
Rebuild after update in master

**Comment by someone235** [comment]
Rebuild after update in master

**Comment by someone235** [comment]
Rebuild after update in master

**Pull request merged**